openmetadata-docs/content/deployment/backup-restore-metadata.md

```diff
@@ -152,6 +152,7 @@ An example of S3 CLI call will look as:
 ```commandline
 metadata backup -u openmetadata_user -p openmetadata_password \
     -H localhost -d openmetadata_db --output=dir1/dir2 \
+    --upload-destination-type AWS \
```

Comment:
can we make AWS as default upload destination in the code itself, thoughts?

Reply:
I think we don't have to do that because it's optional parm.